src/Tizen.NUI.Components/Controls/Progress.cs

```diff
@@ -127,10 +128,13 @@ public class Progress : Control
         private ImageView trackImage = null;
         private ImageView progressImage = null;
         private ImageView bufferImage = null;
+        private ImageVisual indeterminateImage = null;
         private float maxValue = 100;
         private float minValue = 0;
         private float currentValue = 0;
         private float bufferValue = 0;
+        private Animation indeterminateAnimation = null;
+        private ProgressStyle progressStyle => ViewStyle as ProgressStyle;
```

Comment:
`progressStyle` is not used in the code. Could you remove the variable?

Reply:
Okay, I'll remove it.